app/views/stream_entries/_simple_status.html.haml

```diff
@@ -15,6 +15,7 @@
         %span.display-name
           %bdi
             %strong.display-name__html.p-name.emojify= display_name(status.account, custom_emojify: true, autoplay: autoplay)
+          &nbsp;
```

Comment:
This `&nbsp;` forces that there always be at least one space character between display name and the account address, so I can accept is as a good-enough solution.

However, for **L**TR styles, it'll cause _two_ spaces between name and address. Moreover, for RTL styles, the original space at the right side of Display name would remain.

I don't know how to fix the above two (minor) problems. But I am already happy with your solution. Thanks!